### ISSUE TITLE
Use upstream `org.reflections`

### ISSUE THREAD

**Issue by BenjaminAmos** [MovingBlocks/gestalt#132]
Currently gestalt depends on a custom snapshot of what was `org.reflections:0.9.12`, re-named to `org.terasology:reflections:0.9.12-MB`. I do not know the exact reasons for the fork but [MovingBlocks/reflections](https://github.com/MovingBlocks/reflections) only contains a single unique commit with minor code changes. The README claims that the project is unsupported but things appear to have changed since then, with several upstream `org.reflections` releases being made, up to the most recent [`org.reflections:0.10.2`](https://github.com/ronmamo/reflections/releases/tag/0.10.2). It might be worth upgrading to the upstream source to benefit from any bugfixes that might have occurred since then.

This is mostly an obsolete issue with gestalt 8 (`develop` branch), since it doesn't rely on reflections anymore. Terasology and Destination Sol could still be relying on gestalt 7 for some time to come though.

**Comment by keturn** [comment]
I'm not finding all my notes, but here's what I remember of the last time I looked at this:

At the time, I think there had been some newer reflections release, at least 0.9.12. But looking over the bug reports, it was clear there were some problems, and trading my then-problems for _different_ problems didn't seem worth it.

If that was sometime around https://github.com/MovingBlocks/Terasology/pull/4564 , then it was before the upstream 0.10.x releases came out. Maybe those are better?

I think by then (October '21), it was long enough after the v8 work had started that I hoped v8 would be released before I had to worry about Reflections versions again.

**Comment by soloturn** [comment]
they use gestalt-8 now isn't it @BenjaminAmos and this can be closed?

**Comment by BenjaminAmos** [comment]
That is correct: usage of the `org.reflections` library was removed in Gestalt 8.
